examples/gazebo_scene_viewer/SceneManager.cc

```diff
@@ -1833,12 +1835,13 @@ void SubSceneManager::CreateGeometryFunctionMap()
   this->geomFunctions[gazebo::msgs::Geometry::BOX] =
       &SubSceneManager::ProcessBox;
 
-  // TODO(anyone): enable when cone protobuf msg created
+  // todo(anyone): enable when cone protobuf msg is created
```

Comment:
The Cone msg is there too https://github.com/ignitionrobotics/ign-msgs/blob/main/proto/ignition/msgs/geometry.proto#L74